js/views/inbox_view.js

```diff
@@ -49,14 +49,13 @@
       if (!e.ctrlKey) {
         return;
       }
-      const keyCode = e.which || e.keyCode;
       const maxSize = 22; // if bigger text goes outside send-message textarea
       const minSize = 14;
-      if (keyCode === 189 || keyCode === 109) {
```

Comment:
It was two different keycodes before, and now it's one. Why?

Reply:
its because the `-` key is in two spots on the keyboard and has 2 different keys. Same with equals below.